modules/UI/toolbars/Toolbar.js

```diff
@@ -219,7 +219,10 @@ const defaultToolbarButtons = {
         id: 'toolbar_contact_list',
         tooltipKey: 'bottomtoolbar.contactlist',
         className: 'button icon-contactList',
-        sideContainerId: 'contacts_container'
+        sideContainerId: 'contacts_container',
+        html: `<span class="badge-round">
```

Comment:
Probably you wanted to put this:
`<span class="badge-round">
                    <span id="numberOfParticipants"></span>
                </span>`